tests/manage/noobaa/test_write_to_bucket.py

```diff
@@ -0,0 +1,61 @@
+import logging
+
+import boto3
+import pytest
+
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import ManageTest, tier1
+from ocs_ci.ocs import constants
+from tests.helpers import create_unique_resource_name
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.skipif(condition=True, reason="NooBaa is not deployed")
+@pytest.mark.skipif(
+    condition=config.ENV_DATA['platform'] != 'AWS',
+    reason="Tests are not running on AWS deployed cluster"
+)
+@tier1
+class TestBucketIO(ManageTest):
+    """
+    Test IO of a bucket
+    """
+    @pytest.mark.polarion_id("OCS-472")
+    def test_write_file_to_bucket(self, noobaa_obj, awscli_pod, created_buckets, uploaded_objects):
+        """
+        Test object IO using the S3 SDK
+        """
+        base_command = (
```

Comment:
please move to constants/create a function

Reply:
Moved to `helpers/craft_s3_command()`